cmd/greeter_en/BUILD

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# This target allows us to use `./pants run` and `./pants package` on this `main` Go package.
+#
+# You can optionally set the field `output_path="greeter_en"`, for example, for the binary's name
+# to be different when running `./pants package`.
+go_binary(
+    name="bin",
```

Comment:
The output is going to be `./dist/cmd.greeter_en/bin`. Go users expect the binary name to be the name of the diectory, this `greeter_en`. Maybe add `output_path` or change the `name`?

Reply:
Good idea. We lose a guarantee that the name is globally unique, but seems reasonable to change